docs/docs/troubleshooting.md

```diff
@@ -12,14 +12,22 @@ Please check the [list of frequently asked questions](faq.md).
 
 * Surround Sound isn't supported in xemu yet, you should disable this in your [EEPROM](eeprom.md) if you have enabled it.
 
-## Flatpak release will not load BIOS / MCPX ROM / Hard Drive
+## Flatpak release will not load the Hard Drive
 
 Flatpak programs are normally sandboxed, restricting their access to the filesystem.
 
 By default, xemu can only access `~/.var/app/app.xemu.xemu/data/xemu/xemu`
 
-If you want to allow xemu access to other folders, then:
+To enable write permissions for the hard drive file xbox_hdd.qcow2, you need to grant xemu access to write in a different directory. To proceed, use either flatpak or flatseal, described below:
```

Comment:
That's not necessary per-se if the file is at `~/.var/app/app.xemu.xemu/data/xemu/xemu`, which I think it could (should?).
That's explained at https://github.com/flathub/app.xemu.Xemu?tab=readme-ov-file#usage
> Only $HOME/.var/app/app.xemu.xemu/data/xemu/xemu can be written by xemu. The Hard Disk image has to be placed there, for example, at $HOME/.var/app/app.xemu.xemu/data/xemu/xemu/xbox_hdd.qcow2.